.github/workflows/pushdocs.yml

```diff
@@ -0,0 +1,28 @@
+name: pushdocs.yml
+
+on:
+    push:
+        branches:
+            - attackspeed
+            - gh-pages
+
+jobs:
+    deploy:
+        name: Deploy to github pages
+        runs-on: ubuntu-latest
+        steps:
+        - uses: action/chechout@attackspeed
+
+        - run: sudo apt-get install doxygen
+        - run: sudo apt-get install graphviz
+        - run: doxygen doxconfig.txt
```

Comment:
A configot a doxygen-nel generáld ki. Ha azzal generálod akkor annak nincs kiterjesztése. Ez itt bizony egy .txt.

Reply:
Ezt a doxigen generálta ki, csak én windowst használok és ott ilyet generál.

Reply:
Ha az megoldja a problemat akkor leszedem a .txt kiterjesztést róla, de kicsit másként is nézett ki, mint a videóban a tiéd.

Reply:
Nyilván ha windows alatt generálod ki akkor nem feltétlen lesz jó a config, ha azt ununtu-n pusholod a workflowban. 
Próbáld meg WSL alatt, és mennie kell.

Reply:
@vizvezetek fent van az új config, de a workflow még így is ugyan azt a hibaüzenetet dobja: 

Check failure
GitHub Actions
/ .github/workflows/pushdocs.yml
Error
every step must define a `uses` or `run` key

Reply:
Használd bátran a Google-t és hamar meglesz a hiba.